languageFactories/language-factory-core/src/test/scala/cromwell/languages/util/ImportResolverSpec.scala

```diff
@@ -34,14 +34,14 @@ class ImportResolverSpec extends FlatSpec with Matchers {
   }
 
   it should "resolve a path from no initial root" in {
-    val resolver = HttpResolver()
+    val resolver = HttpResolver(NoopResolvedImportsStore)
```

Comment:
Another case where a default value of `NoopResolvedImportsStore` could avoid introducing a new concept into this area of the code (I think this is especially valid because the noop... does nothing)